{
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [
      {
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:a:connman_project:connman:*:*:*:*:*:*:*:*",
            "versionEndIncluding": "1.34",
            "vulnerable": true
          }
        ],
        "operator": "OR"
      },
      {
        "cpe_match": [
          {
            "cpe23Uri": "cpe:2.3:o:debian:debian_linux:8.0:*:*:*:*:*:*:*",
            "vulnerable": true
          }
        ],
        "operator": "OR"
      }
    ]
  },
  "cve": {
    "CVE_data_meta": {
      "ASSIGNER": "cve@mitre.org",
      "ID": "CVE-2017-12865"
    },
    "affects": {
      "vendor": {
        "vendor_data": [
          {
            "product": {
              "product_data": [
                {
                  "product_name": "connman",
                  "version": {
                    "version_data": [
                      {
                        "version_affected": "\u003c=",
                        "version_value": "1.34"
                      }
                    ]
                  }
                }
              ]
            },
            "vendor_name": "connman_project"
          },
          {
            "product": {
              "product_data": [
                {
                  "product_name": "debian_linux",
                  "version": {
                    "version_data": [
                      {
                        "version_affected": "=",
                        "version_value": "8.0"
                      }
                    ]
                  }
                }
              ]
            },
            "vendor_name": "debian"
          }
        ]
      }
    },
    "data_format": "MITRE",
    "data_type": "CVE",
    "data_version": "4.0",
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "Stack-based buffer overflow in \"dnsproxy.c\" in connman 1.34 and earlier allows remote attackers to cause a denial of service (crash) or execute arbitrary code via a crafted response query string passed to the \"name\" variable."
        }
      ]
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [
            {
              "lang": "en",
              "value": "CWE-119"
            }
          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "name": "DSA-3956",
          "refsource": "DEBIAN",
          "tags": [
            "Third Party Advisory"
          ],
          "url": "http://www.debian.org/security/2017/dsa-3956"
        },
        {
          "name": "100498",
          "refsource": "BID",
          "tags": [
            "Third Party Advisory",
            "VDB Entry"
          ],
          "url": "http://www.securityfocus.com/bid/100498"
        },
        {
          "name": "https://01.org/security/intel-oss-10001/intel-oss-10001",
          "refsource": "CONFIRM",
          "tags": [],
          "url": "https://01.org/security/intel-oss-10001/intel-oss-10001"
        },
        {
          "name": "https://bugzilla.redhat.com/show_bug.cgi?id=1483720",
          "refsource": "CONFIRM",
          "tags": [
            "Issue Tracking",
            "Patch",
            "Third Party Advisory"
          ],
          "url": "https://bugzilla.redhat.com/show_bug.cgi?id=1483720"
        },
        {
          "name": "https://git.kernel.org/pub/scm/network/connman/connman.git/commit/?id=5c281d182ecdd0a424b64f7698f32467f8f67b71",
          "refsource": "CONFIRM",
          "tags": [
            "Issue Tracking",
            "Patch",
            "Third Party Advisory"
          ],
          "url": "https://git.kernel.org/pub/scm/network/connman/connman.git/commit/?id=5c281d182ecdd0a424b64f7698f32467f8f67b71"
        },
        {
          "name": "GLSA-201812-02",
          "refsource": "GENTOO",
          "tags": [],
          "url": "https://security.gentoo.org/glsa/201812-02"
        },
        {
          "name": "https://www.nri-secure.com/blog/new-iot-vulnerability-connmando",
          "refsource": "MISC",
          "tags": [],
          "url": "https://www.nri-secure.com/blog/new-iot-vulnerability-connmando"
        }
      ]
    }
  },
  "impact": {
    "baseMetricV2": {
      "cvssV2": {
        "accessComplexity": "LOW",
        "accessVector": "NETWORK",
        "authentication": "NONE",
        "availabilityImpact": "PARTIAL",
        "baseScore": 7.5,
        "confidentialityImpact": "PARTIAL",
        "integrityImpact": "PARTIAL",
        "vectorString": "AV:N/AC:L/Au:N/C:P/I:P/A:P",
        "version": "2.0"
      },
      "exploitabilityScore": 10,
      "impactScore": 6.4,
      "obtainAllPrivilege": false,
      "obtainOtherPrivilege": false,
      "obtainUserPrivilege": false,
      "severity": "HIGH",
      "userInteractionRequired": false
    },
    "baseMetricV3": {
      "cvssV3": {
        "attackComplexity": "LOW",
        "attackVector": "NETWORK",
        "availabilityImpact": "HIGH",
        "baseScore": 9.8,
        "baseSeverity": "CRITICAL",
        "confidentialityImpact": "HIGH",
        "integrityImpact": "HIGH",
        "privilegesRequired": "NONE",
        "scope": "UNCHANGED",
        "userInteraction": "NONE",
        "vectorString": "CVSS:3.0/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H",
        "version": "3.0"
      },
      "exploitabilityScore": 3.9,
      "impactScore": 5.9
    }
  },
  "lastModifiedDate": "2018-12-03T11:29Z",
  "publishedDate": "2017-08-29T16:29Z"
}